build.gradle

```diff
@@ -4,6 +4,7 @@ plugins {
     id 'signing'
     id 'org.spongepowered.plugin' version '0.8.1'
     id 'net.minecrell.licenser' version '0.3'
+    id "com.github.johnrengelman.shadow" version "1.2.3"
```

Comment:
Change this to shadow version 1.2.4 as I believe that is out, unless there is a reason you used 1.2.3.